Source/WebKit/gtk/migrating-to-webkitgtk-6.0.md

```diff
@@ -28,7 +28,11 @@ JavaScriptCore API (e.g. [type@JSC.Context] and [type@JSC.Object]) that is
 available since 2.22. It also includes the entire GObject DOM API (e.g.
 `WebKitDOMDocument`), which has been removed without replacement. Use JavaScript
 to interact with and manipulate the DOM instead, perhaps via
-[method@WebKit.WebView.run_javascript] or [method@JSC.ValueObject.invoke_method].
+[method@WebKit.WebView.evaluate_javascript] or
+[method@WebKit.WebView.call_async_javascript_function].
```

Comment:
This is not correct, we are mixing ui and web process APIs here. `WebKitDOMDocument` is web process API, so the replacement would be the JSC GLib API. For example, to get the document you can:

```
JSCContext *jsContext = webkit_frame_get_js_context (frame);
JSCValue *jsDocument = jsc_context_get_value (jsContext, "document");
```

Reply:
Updated this a bit to clarify which process the APIs can be used in. Do you want me to additionally add that example? If so, I'll probably split it out into a separate section to keep this section small and focused.

Reply:
Well, I see you approved this PR, so I'll land now and we can follow up if desired.